Cargo.lock

```diff
@@ -9697,14 +9803,59 @@ dependencies = [
 name = "test-parachains"
 version = "0.8.30"
 dependencies = [
- "parity-scale-codec",
+ "parity-scale-codec 2.0.0",
  "polkadot-parachain",
  "sp-core",
  "test-parachain-adder",
  "test-parachain-halt",
  "tiny-keccak",
 ]
 
+[[package]]
+name = "test-runner"
+version = "0.9.0"
+source = "git+https://github.com/paritytech/substrate?branch=master#2e34d473e6ba81b454eb039544f2162975c0b30b"
+dependencies = [
+ "env_logger 0.7.1",
+ "frame-system",
+ "futures 0.1.29",
+ "futures 0.3.13",
+ "jsonrpc-core",
+ "log",
+ "parity-scale-codec 1.3.7",
```

Comment:
why does it depend on parity-scale-codec v1 and not v2?

Reply:
ahh, must be vestigial, can quickly bump it

Reply:
please address https://github.com/paritytech/substrate/pull/8485#discussion_r603587200 before you do